pandas/core/ops.py

```diff
@@ -1568,7 +1562,7 @@ def wrapper(self, other):
                 dtype = getattr(other, 'dtype', None)
                 other = SparseArray(other, fill_value=self.fill_value,
                                     dtype=dtype)
-            return _sparse_array_op(self, other, op, name)
+            return _sparse_array_op(self, other, op, name, series=False)
```

Comment:
this whole sparse interaction is pretty hacky (I know not addressing this now), but....

Reply:
Yah, there are a couple of improvements in store for the pass after next (which is just implementing the single-call `ops.add_whatever_methods(cls)`)